Mathlib/CategoryTheory/MorphismProperty/Comma.lean

```diff
@@ -50,6 +50,18 @@ lemma over_iso_iff (P : MorphismProperty T) [P.RespectsIso] {X : T} {f g : Over
     P f.hom ↔ P g.hom :=
   P.comma_iso_iff e
 
+/-- The morphism property on `Over X` induced by a morphism property on `C`. -/
+def over (W : MorphismProperty T) {X : T} : MorphismProperty (Over X) := fun _ _ f ↦ W f.left
```

Comment:
It is unfortunate we cannot do this automatically, but could you add `simp` lemmas `over_iff`, etc.